libparsec/crates/types/schema/local_device/device_file_keyring.json5

```diff
@@ -0,0 +1,30 @@
+{
+    "label": "DeviceFileKeyring",
+    "type": "keyring",
+    "other_fields": [
+        {
+            "name": "ciphertext",
+            "type": "Bytes"
+        },
+        {
+            "name": "human_handle",
+            "type": "HumanHandle"
+        },
+        {
+            "name": "device_label",
+            "type": "DeviceLabel"
+        },
+        {
+            "name": "device_id",
+            "type": "DeviceID"
+        },
+        {
+            "name": "organization_id",
+            "type": "OrganizationID"
+        },
+        {
+            "name": "slug",
```

Comment:
@vxgmichel shouldn't there be an additional ID here to retrieve the key in the keyring ?

Reply:
:+1: